guac_actix/src/lib.rs

```diff
@@ -79,17 +108,17 @@ impl Message for Tick {
 impl Handler<Tick> for PaymentController {
```

Comment:
are we spawning a tick progress for each counterparty here?

Reply:
yes